code/modules/reagents/chemistry/holder/holder.dm

```diff
@@ -504,6 +505,7 @@
 		log_target.add_hiddenprint(transferred_by) //log prints so admins can figure out who touched it last.
 		log_combat(transferred_by, log_target, "transferred reagents to", my_atom, "which had [get_external_reagent_log_string(transfer_log)]")
 
+	playsound(my_atom.loc, pick(transfer_sound), 50)
```

Comment:
Hell no. This proc is very hot and is called by plumbing factories A LOT. I can't even imagine how painful large plumbing factories are goanna sound.

Move this to sound to play when just transferring reagents between beakers and such